Polygon_mesh_processing/examples/Polygon_mesh_processing/detect_features_example.cpp

```diff
@@ -27,15 +27,25 @@ int main(int argc, char* argv[])
   typedef boost::property_map<Mesh, CGAL::edge_is_feature_t>::type EIFMap;
   typedef boost::property_map<Mesh, CGAL::face_patch_id_t<int> >::type PIMap;
   typedef boost::property_map<Mesh, CGAL::vertex_incident_patches_t<int> >::type VIMap;
+  typedef boost::property_map<Mesh, CGAL::vertex_is_constrained_map>::type VCMap
 
   EIFMap eif = get(CGAL::edge_is_feature, mesh);
   PIMap pid = get(CGAL::face_patch_id_t<int>(), mesh);
   VIMap vip = get(CGAL::vertex_incident_patches_t<int>(), mesh);
+  VCMap vcmap =get(CGAL::vertex_is_constrained_map,meshj);
```

Comment:
Does this line compile?

Reply:
I made an error. I fixed it just now!

Reply:
And please add the whitespace right after the `=` and the `,` just as it is done in the line above.    This is not done everywhere, but adapt to the local style of a file.

Reply:
> 
> Does this line compile?

So the general rule (which I unfortunately also too often do not respect) is to compile/run the testsuite of the package you modify, before a git push.

Reply:
Okay, I'll compile it, fix all the errors and get back!